code/modules/projectiles/guns/projectile/constructable/blinder.dm

```diff
@@ -0,0 +1,193 @@
+	icon_state = "polaroid"
+	item_state = "polaroid"
+	w_class = 2.0
+	flags = FPRINT
+	siemens_coefficient = 1
+	slot_flags = SLOT_BELT
+	origin_tech = "materials=1;engineering=1"
+	starting_materials = list(MAT_IRON = 2000)
+	w_type = RECYK_ELECTRONIC
+	var/cell = null
+	var/bulb = 1
+	var/burnedout = 0
+	var/powercost = 10000
+
+/obj/item/device/blinder/Destroy()
+	if(cell)
+		qdel(cell)
+		cell = null
+	..()
+
+/obj/item/device/blinder/New()
+	..()
+	update_verbs()
+
+/obj/item/device/blinder/examine(mob/user)
+	..()
+	if(bulb && burnedout)
+		to_chat(user, "<span class='warning'>\The [src]'s flash bulb is broken.</span>")
+	else if (!bulb)
+		to_chat(user, "<span class='info'>\The [src] appears to be missing a flash bulb.</span>")
+
+/obj/item/device/blinder/attack_self(mob/user as mob)
+	if(!bulb || burnedout || !cell)
+		if (user)
+			user.visible_message("*click click*", "<span class='danger'>*click*</span>")
+			playsound(user, 'sound/weapons/empty.ogg', 100, 1)
+		else
+			src.visible_message("*click click*")
+			playsound(get_turf(src), 'sound/weapons/empty.ogg', 100, 1)
+		return
+
+	if(cell)
+		var/obj/item/weapon/cell/C = cell
+		if(C.charge < powercost)
+			user.visible_message("[user] presses the button on \the [src], but the flash bulb merely flickers.","You press the button on \the [src], but the flash bulb merely flickers.")
+			to_chat(user, "<span class='warning'>There's not enough energy in the cell to power the flash bulb!</span>")
+			playsound(get_turf(src), 'sound/weapons/empty.ogg', 100, 1)
+			return
+
+		var/flash_turf = get_turf(src)
+		if(!flash_turf)
+			return
+		for(var/mob/living/M in get_hearers_in_view(7, flash_turf))
+			flash(get_turf(M), M)
+
+
+		user.visible_message("<span class='danger'>[user] overloads \the [src]'s flash bulb!</span>","<span class='danger'>You overload \the [src]'s flash bulb!</span>")
+		to_chat(user, "<span class='warning'>\The [src]'s flash bulb shatters!</span>")
+
+		C.charge -= powercost
+
+		burnedout = 1
+		update_verbs()
+
+/obj/item/device/blinder/proc/flash(var/turf/T , var/mob/living/M)
+	playsound(get_turf(src), 'sound/weapons/flash.ogg', 100, 1)
+
+	var/eye_safety = 0
+	if(iscarbon(M))
+		var/mob/living/carbon/C = M
+		eye_safety = C.eyecheck()
+
+	if(eye_safety < 1)
+		flick("e_flash", M.flash)
+
+	if(issilicon(M))
+		M.Weaken(rand(5, 10))
+		M.visible_message("<span class='warning'>[M]'s sensors are overloaded by the flash of light!</span>","<span class='warning'>Your sensors are overloaded by the flash of light!</span>")
+
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		var/datum/organ/internal/eyes/E = H.internal_organs_by_name["eyes"]
+		if (E && E.damage >= E.min_bruised_damage)
+			to_chat(M, "<span class='warning'>Your eyes start to burn badly!</span>")
+	M.update_icons()
+
+/obj/item/device/blinder/proc/update_verbs()
+	if(cell)
+		verbs += /obj/item/device/blinder/verb/remove_cell
```

Comment:
Did you make sure this works? Last I checked you cant actually ever remove a verb from verbs.

The only thing you can add/remove to verbs are normal procs